Add EC science duty gloves + Science bowman + Bowman sprite changes

Ryan180602 opened the pull request:
:cl: Ryan180602
rscadd: Adds EC Science Duty Gloves.
rscadd: Adds EC Science Bowman Headset.
tweak: Makes most bowman headsets have antenna on .N-S onmob sprite.
/:cl:

~~The bowmans have antennas on the E-W sprites, and item sprites. But not N-S.~~
Those that have antenna on item sprites, have it on on-mobs now.

Also adds science duty gloves and bowman. Not sure why they weren't a thing.
![GM0n8Mu](https://user-images.githubusercontent.com/50071611/174652759-a38c5315-9e30-47d2-a9d5-944d7b40c438.png)

Spookerton commented on maps/torch/items/encryption_keys.dm:
```diff
@@ -3,8 +3,8 @@
 
-/obj/item/device/encryptionkey/headset_torchnt
-	name = "corporate radio encryption key"
+/obj/item/device/encryptionkey/headset_torch
```
```suggestion
/obj/item/device/encryptionkey/science
```

Spookerton commented on maps/torch/items/headsets.dm:
```diff
@@ -1,9 +1,15 @@
 	icon_state = "sci_headset"
 	item_state = "headset"
-	ks1type = /obj/item/device/encryptionkey/headset_torchnt
+	ks1type = /obj/item/device/encryptionkey/headset_torch
```
```suggestion
	ks1type = /obj/item/device/encryptionkey/science
```

SierraKomodo commented on maps/torch/outfits/scgec.dm:
```diff
@@ -226,7 +226,7 @@
+		/obj/item/storage/box/large/scgec_gloves,
 		/obj/item/storage/box/scgec_department,
 		/obj/item/storage/box/scgec_department_service
 	)
```
Test failure:

```
 ## UNIT_TEST ##: --------------- dressup accessories, SCGEC [/obj/item/storage/backpack/dufflebag/scgec_accessories] Contains more items than it has storage space for (44 / 40). 
## UNIT_TEST ##: !!! FAILURE !!! [EQUIPMENT: Storage items should be able to actually hold their initial contents]: [1] Some storage item types were not able to hold their default initial contents.
```

You can resolve this by updating the `max_storage_space` var below to `44`.

SierraKomodo reviewed (dismissed):
- The item sprite for the research bowman has an antenna, but the on-mob sprite lacks an antenna.
- The item sprite for the other bowmans lack an antenna, but the on-mob sprite has an antenna.
- Looking at the pre existing on-mob sprites, I don't think the W/E sprites have antenna - I think that vertical line is meant to be the headband for the bowman headset. Note how it's present even on the side where the antenna would have been on the opposide side of the head and thereby not visible.

Ryan180602 commented:
> * The item sprite for the research bowman has an antenna, but the on-mob sprite lacks an antenna.
> * The item sprite for the other bowmans lack an antenna, but the on-mob sprite has an antenna.
> * Looking at the pre existing on-mob sprites, I don't think the W/E sprites have antenna - I think that vertical line is meant to be the headband for the bowman headset. Note how it's present even on the side where the antenna would have been on the opposide side of the head and thereby not visible.

Fixed one and two.
Also removed the onmob antennaes. You're right, I'm dum.

SierraKomodo reviewed (dismissed):
- Duty glove on-mob sprites aren't aligned with the hands: https://user-images.githubusercontent.com/11140088/174684633-8443fd9c-f855-46d9-b87f-075b64d85a57.mp4
- Research bowman on-mob doesn't match the other bowmans - The headband is cut off: ![dreamseeker_2022-06-20_18-32-59](https://user-images.githubusercontent.com/11140088/174684950-19cd3631-d39f-48af-896b-5b9b67a9be9f.png) vs ![dreamseeker_2022-06-20_18-32-01](https://user-images.githubusercontent.com/11140088/174684955-c33f91a4-fe15-404c-a69e-ede2c7ce0f4c.png)
- Research bowman object sprite doesn't match the other bowmans - The main body is narrower: ![dreamseeker_2022-06-20_18-32-54](https://user-images.githubusercontent.com/11140088/174684986-1ba25a17-c259-47b0-93f1-63bb145be201.png)
-

Ryan180602 committed:
add EC science duty gloves + science bowman

also gives bowman headsets antenna in N-S sprites
a

Ryan180602 commented:
fixed back the gloves, i made a lapse in judgement and thought it was misaligned *before*.
fixed the headband
made the exploration bowman, miner bowman and science bowman similar to the rest item-sprite wise.

i shouldve caught those, am sorry
